### ISSUE TITLE
Error in schema validation for ItemBatcher

### ISSUE THREAD

**Issue by tvhees** [ChristopheBougere/asl-validator#130]
**Describe the bug**
Error when validating DISTRIBUTED mode Map step with ItemBatcher option specified.

Error:
✕ State machine "batchedMachine" definition is invalid:
SCHEMA_VALIDATION_FAILED: /States/Batched/ItemBatcher is invalid. must match exactly one schema in oneOf

**To Reproduce**
Please provide a complete state machine definition:
```yaml
    batchedMachine:
      name: 'batchedMachine'
      definition:
        Comment: 'batched distributed map step function'
        StartAt: Batched
        States:
          Batched:
            Type: Map
            MaxConcurrency: 1
            ItemsPath: $.items
            ResultPath: $.batched
            End: true
            ItemBatcher:
              MaxItemsPerBatch: 2
              BatchInput:
                batchInput: inputValue
            ItemProcessor:
              ProcessorConfig:
                Mode: DISTRIBUTED
                ExecutionType: EXPRESS
              StartAt: Test-Batched
              States:
                Test-Batched:
                  Type: Pass
                  End: true
```

**Expected behavior**
State machine definition should pass validation

**Version:**
node 16.11.1
serverless: 3.28.1
SDK: 4.3.2
serverless-step-functions: ^3.13.0

**Comment by tvhees** [comment]
I believe the current validation schema is not specific enough, causing valid `ItemBatcher` configurations to match both entries in the `oneOf` case.

There are several combinations allowed for this feature by AWS. There are two paired properties:
MaxItemsPerBatch + MaxItemsPerBatchPath
MaxInputBytesPerBatch + MaxInputBytesPerBatchPath

You must specify at least one of the four properties. You may specify one property from each pair. You may not specify both properties from a single pair.

**Comment by tvhees** [comment]
Just for clarity - I closed the automatically linked PR and created a cleaner one with the suggested solution:
https://github.com/ChristopheBougere/asl-validator/pull/134

**Comment by ChristopheBougere** [comment]
Released in 3.6.1
Thanks for your contribution!
